statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -490,14 +474,17 @@ def __init__(self, endog, exog, groups, time=None, family=None,
         self.cov_struct = cov_struct
 
         # Handle the offset and exposure
-        self._offset_exposure = np.zeros(len(self.endog))
+        self._offset_exposure = None
         if offset is not None:
-            self._offset_exposure += self.offset
+            self._offset_exposure = self.offset.copy()
```

Comment:
ignore it, self._offset_exposure will/might be modified again later, we need copy